src/components/NavSpace.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts" setup>
+import { computed } from 'vue';
+import { useRoute } from 'vue-router';
+import { useUiStore } from '@/stores/ui';
+import { useTreasury } from '@/composables/useTreasury';
+import { useSpacesStore } from '@/stores/spaces';
+
+import IHGlobeAlt from '~icons/heroicons-outline/cash';
+import IHNewspaper from '~icons/heroicons-outline/newspaper';
+import IHCash from '~icons/heroicons-outline/cash';
+import IHCog from '~icons/heroicons-outline/cog';
+
+const route = useRoute();
+const uiStore = useUiStore();
+const spacesStore = useSpacesStore();
+
+// TODO if we not get ALL spaces like paginate 1000 we will not find a space here
```

Comment:
I'm not sure I understand, it's not supposed to get the space from listing all spaces, but when it's fetched at Space view:
https://github.com/snapshot-labs/sx-ui/blob/07233f60a2fa9138ff362633a0a560ab9af7ed3c/src/views/Space.vue#L14-L16

Reply:
We want to know if space got a treasury in NavSpace so thats why we try to find it in all spaces

Reply:
ok and it will find it cos fetchSpace() adds space to networksMap.spaces